### DISCUSSION TITLE
Codespaces generates: bash: [: missing `]'

### DISCUSSION THREAD

**Discussion by potatoqualitee** [community/community#51937] [Codespaces]
### Select Topic Area

Bug

### Body

When I first boot a machine, it generates a shell with bash: [: missing \`]'. I don't think this is mine because I use powershell as my shell, but maybe i forgot. Anyway, I traced it down to ` /root/.bashrc`

root@b52a4cda8bee:/workspace# . /root/.bashrc
bash: [: missing `]'

Though that's all I know about it

**Comment by potatoqualitee** [comment]
Initially, I deleted my codespace, tried again and still got the same error. Then I deleted `. /root/.bashrc`, deleted my codespace and tried again and the error was gone (seems a new .bashrc was generated too)

**Comment by aishraj** [comment]
We are happy you are here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction. Does your repository use a custom devcontainer?

**Reply by potatoqualitee** [reply]
Yes, I was reporting it moreso if it is an issue with GitHub's runners. I have solved my own issue. We are using devcontainers and we don't mess with bash (tmk), as it's a PowerShell project.

https://github.com/dataplat/dbatools/tree/development/.devcontainer
